### DISCUSSION TITLE
M62-M65

### DISCUSSION THREAD

**Discussion by jeefdumbo** [grblHAL/core#599] [General]
I build a firmware with WebBuilder for T41U5XBB Board with 4 Axis and Y Autosquared
When i Control the Aux outputs with M62P0 or M64P0 M63P0 or M65P0 i get the error 36: unused value words found in block
How can i Turn on or Off the Aux Ports?

**Comment by phil-barrett** [comment]
Hi, sorry the late response, I don't always scan the discussions section.  The accepted form is G6[2345] P0.  Is it possible you used an O (letter) instead of a 0 (number)?

**Reply by terjeio** [reply]
The issue was due to a regression (partial modal group change), fixed now.
